front_end/readme.md

```diff
@@ -0,0 +1,36 @@
+# Readme 
+
+This project is based on [react-redux-scaffold](https://github.com/lcjnil/react-redux-scaffold).
+
+Write in progress, issues are welcomed.
+
+## Config
+
+Modify `settings.conf.js` variables `devApi` or `liveApi` with your API url. 
+
+## Run
+
+`npm install` to install dependencies.
+
+`gulp serve` to start a local server with hot reloading.
+
+`gulp build:dist` to build production code in the /dist directory.
```

Comment:
I did the `npm install` but can't get gulp running:

```
[rlr (sp2) front_end]$ gulp
[13:30:58] No gulpfile found
```

I'll file an issue for this.